org.mersenne.mprime.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.mersenne.mprime</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>EULA</project_license>
```

Comment:
Does the license allow redistribution?

Reply:
Skimming through the EULA, I don't see any immediate clause regarding redistribution, although I've already contacted the developer of it and the only requirement to meet before submitting it here was to make sure to update it to version 29.8 (current stable one, was a beta version at time of submission).

Reply:
This isn't valid appdata/spdx. It needs to be `LicenseRef-proprietary`